src/sinks/blackhole/config.rs

```diff
@@ -37,6 +37,10 @@ impl SinkConfig for BlackholeConfig {
     fn sink_type(&self) -> &'static str {
         "blackhole"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
I think it can, no? Just simply whenever it consumes the event is when the event has been processed.